packages/flutter_tools/lib/src/runner/flutter_command.dart

```diff
@@ -38,11 +37,15 @@ export '../cache.dart' show DevelopmentArtifact;
 
 abstract class DotEnvRegex {
   // Dot env multi-line block value regex
-  static final RegExp multiLineBlock = RegExp(r'^\s*([a-zA-Z_]+[a-zA-Z0-9_]*)\s*=\s*"""\s*(.*)$');
+  static final RegExp multiLineBlock = RegExp(
```

Comment:
I'm not sure these formatting changes are correct given what's currently at HEAD. Maybe the 100 character limit set in `analysis_options.yaml` isn't being respected properly?